moonbeam-types-bundle/package.json

```diff
@@ -27,6 +28,8 @@
   "dependencies": {
     "@polkadot/keyring": "^7.0.3",
     "@polkadot/types": "^5.2.1",
-    "typescript": "^4.3.5"
+    "@types/underscore": "^1.11.3",
+    "typescript": "^4.3.5",
+    "underscore": "^1.13.1"
```

Comment:
Do we really need underscore for our code ?
If yes, why not import deepEqual instead of re-writing it ?

Reply:
underscore is needed in the check utility. DeepEqual doesn't seem to do what I want it to do. Im going to move these deps to devDeps